lib/msal-browser/docs/errors.md

```diff
@@ -114,8 +114,85 @@ msalInstance.handleRedirectPromise()
 
 If you are using one of our wrapper libraries (React or Angular), please see the error docs in those specific libraries for additional reasons you may be receiving this error:
 
-- [msal-react errors](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-react/docs/errors.md)
-- [msal-angular errors](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/v2-docs/errors.md)
+- [msal-react errors](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-react/docs/errors.md#interaction_in_progress)
+- [msal-angular errors](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/v2-docs/errors.md#interaction_in_progress)
+
+If you are not using any of the wrapper libraries but concerned that your application might trigger concurrent interactive requests, you should check if any other interaction is in progress prior to invoking an interaction in your token acquisition method. You can achieve this by implementing a global application state or a broadcast service etc. that emits the current MSAL interaction status via [MSAL Events API](./events.md).
+
+❌ The following example may throw this error because the `acquireTokenPopup` in the **catch** block does not check if there is another interaction taking place at the moment:
+
+```javascript
+async function myAcquireToken(request) {
+    const msalInstance = getMsalInstance(); // get the msal application instance
+
+    const tokenRequest = {
+        account: msalInstance.getActiveAccount() || null;
+        ...request
+    };
+
+    let tokenResponse;
+
+    try {
+        // attempt silent acquisition first
+        tokenResponse = await msalInstance.acquireTokenSilent(request);
+    } catch (error) {
+        if (error instanceof InteractionRequiredAuthError) {
+            try {
+                tokenResponse = await msalInstance.acquireTokenPopup(request);
```

Comment:
Would be good if we can somehow show an example which calls `acquireTokenPopup` twice to really drive this home. Perhaps at the bottom we can call `myAcquireToken` twice without waiting?

Reply:
done